src/client/Client.cc

```diff
@@ -14036,6 +14036,8 @@ int Client::_rename(Inode *fromdir, const char *fromname, Inode *todir, const ch
     else
       return -CEPHFS_EROFS;
   }
+
+  // don't allow cross-quota renames
```

Comment:
@vshankar 
There's no changes on this PR except for these two line. Should this backport be still merged?

Reply:
Sorry, I didn't understand what you mean here.

Reply:
I left a comment on the pacific backport - https://github.com/ceph/ceph/pull/52580/files#r1274358317